index.html

```diff
@@ -4292,6 +4306,493 @@ <h3>Proof Formats</h3>
+following fields: <var>status</var> to `false`, <var>warnings</var> to an
+empty array, and <var>errors</var> to an empty array.
+          </li>
+          <li>
+Determine the specific mechanism for extracting the <a>conforming document</a>
+(<var>extractDocument</var>):
+            <ol class="algorithm">
+              <li>
+If the <var>inputMediaType</var> is `application/vc+ld+json` or
+`application/vp+ld+json`, the <var>extractDocument</var> function takes
+<var>inputBytes</var> as an input parameter, parses the value as JSON into
+an object, removes the `proof` property, and returns the result as an object.
+If JSON parsing fails, a <a href="#PARSING_ERROR">PARSING_ERROR</a> is appended
+to <var>securingMechanismVerified</var>.<var>errors</var>.
+              </li>
+              <li>
+If the <var>inputMediaType</var> is associated with the JOSE or COSE
+specifications, the [[?VC-JOSE-COSE]] specification defines a process for
+extracting the <a>conforming document</a>.
+<div class="issue" title="Document extraction process in vc-jose-cose">
+The process for extracting a conforming document is not clearly documented in
+the [[?VC-JOSE-COSE]] specification yet, but once it is, a link to that section
+of the specification will be included in this text.
+</div>
+              </li>
+              <li>
+Other extracting functions are possible, but need not be listed in
+the <a data-cite="?VC-SPECS#securing-mechanisms">Securing Mechanisms</a>
+section of the Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+Implementers might find these other specifications through other standards
+bodies or communities that utilize this specification.
+              </li>
+            </ol>
+          </li>
+          <li>
+Determine the specific cryptographic verification algorithm
+(<var>verifyProof</var>) required to verify the securing mechanism:
+            <ol class="algorithm">
+              <li>
+If the <var>inputMediaType</var> is `application/vc+ld+json` or
+`application/vp+ld+json`, the Verifiable Credential Data Integrity
+Specification [[?VC-DATA-INTEGRITY]] defines the
+<a data-cite="VC-DATA-INTEGRITY#verify-proof">process for identifying
+a specific cryptography suite type</a> used for cryptographic verification by
+utilizing the `proof`.`type` property. A list of registered proof types can be
+found in the
+<a data-cite="?VC-SPECS#securing-mechanisms">Securing Mechanisms</a>
+section of the Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+                <ol class="algorithm">
+                  <li>
+Set <var>securedDocument</var> to the result of a JSON parsing function that
+takes <var>inputBytes</var> as an input parameter, parses the value as JSON,
+and returns the result as an object. If JSON parsing fails,
+<var>securingMechanismVerified</var>.<var>status</var> is set to `false`, and a
+<a href="#PARSING_ERROR">PARSING_ERROR</a> is appended to
```

Comment:
If we can get away with not describing JSON parsing, we should... see WebAuthN for additional details: 

https://www.w3.org/TR/webauthn-2/#clientdatajson-serialization

If we do intend to cite serialize and deserialize, the references should be to WHATWG.... 

https://infra.spec.whatwg.org/#serialize-a-javascript-value-to-json-bytes

Reply:
I haven't taken on a dependency on INFRA because you have said that you would "object to taking on a dependency on INFRA because of what happened in the DID WG". Meanwhile, @jyasskin and @iherman were suggesting that we do take on a dependency on INFRA and express the algorithm and variables in terms of INFRA. 

If you have changed your mind, let me know, because I'll be happy to do that if it doesn't lead to you objecting.

Reply:
Its better to use INFRA then it is to redefine JSON Parsing errors.

I suggest not writing extra text, where a citation will do the job.